src/static/facade/rag_poisoning_template.css

```
@@ -0,0 +1,360 @@
+.log-sensitive {
+  color: #ff6b6b;
+  font-weight: bold;
+}
+
+/* Document Checkboxes */
+.rag-analysis-section {
+  margin-top: 20px;
+  padding-top: 16px;
+  border-top: 1px solid #e1e4e8;
+  background: #f6f8fa;
+  padding: 16px;
```

Stylelint (17.10.0)

[error] Overridden property "padding-top" by shorthand "padding" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)